packages/astro/test/preview-routing.test.js

```diff
@@ -1,7 +1,12 @@
 import { expect } from 'chai';
+import { version } from 'vite';
 import { loadFixture } from './test-utils.js';
 
-describe('Preview Routing', () => {
+const IS_VITE_5 = version.startsWith('5.');
+
+// Skip in Vite 5 as it changes how HTML files are served. We may want to review aligning
```

Comment:
Should we add a `TODO` as a reminder in case we need to review them?

Reply:
I've put this on https://linear.app/withastro/issue/PLT-1148/rework-trailingslash-buildformat instead so we don't forget. We should definitely revisit this during Astro 4 though since it will be a breaking change, so I thought a linear issue will help surface and priotitize it.